CHANGELOG

```diff
@@ -43,6 +50,9 @@ Qtile 0.20.0, released 2022-01-24:
           managers to request the name.
         - Fix bug where `Battery` widget did not retrieve `background` from `widget_defaults`.
         - Fix bug where widgets in a `WidgetBox` are rendered on top of bar borders.
+        - Widgets that are incompatible with a backend (e.g. Systray on Wayland) will no longer show 
+          as a ConfigError in the bar. Instead the widget is silently removed from the bar and a message
+          included in the logs.
 
```

Comment:
I think this can be dropped?

Reply:
No. That's exactly what this PR does now (the title here is old)

Reply:
Yep, but the changelog entry is duplicated :smile: 
![image](https://user-images.githubusercontent.com/44536113/160002245-6c7f4099-0173-403d-bb80-ab147f5f167e.png)

Reply:
Whoops!